docs/topics/cache.txt

```diff
@@ -435,8 +450,39 @@ of 60 seconds, and a maximum capacity of 1000 items::
         }
     }
 
-Invalid arguments are silently ignored, as are invalid values of known
-arguments.
+Here, a python-memcached based backend is being configured with an object size
```

Comment:
Use double backticks around python-memcached and pylibmc to fix the spelling issue and avoid having to add them to the wordlist.